src/Dapr.Actors/Runtime/ActorReminder.cs

```diff
@@ -34,13 +34,15 @@ public class ActorReminder : ActorReminderToken, IActorReminder
         /// <param name="state">The state associated with the reminder.</param>
         /// <param name="dueTime">The reminder due time.</param>
         /// <param name="period">The reminder period.</param>
+        /// <param name="ttl">The time at which the reminder will expire.</param>
         public ActorReminder(
             string actorType,
             ActorId actorId,
             string name,
             byte[] state,
             TimeSpan dueTime,
-            TimeSpan period)
+            TimeSpan period,
+            TimeSpan? ttl = null)
```

Comment:
This is also a breaking change.

Reply:
These are the best references I know of if you want to learn more:

- https://github.com/ltrzesniewski/dotnet-runtime/blob/master/docs/coding-guidelines/breaking-changes.md
- https://github.com/ltrzesniewski/dotnet-runtime/blob/master/docs/coding-guidelines/breaking-change-rules.md
- https://github.com/ltrzesniewski/dotnet-runtime/blob/master/docs/coding-guidelines/breaking-change-definitions.md

Reply:
I accidentally linked to someone's fork, but those should all be valid.